<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>222</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="select_function_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>功能选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="func_select_certain_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="data_filter_button">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>51</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>原始表处理</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="func_merge_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>99</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="inv_download_button">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>51</y>
      <width>99</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>库存下载</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="inv_data_process_button">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>80</y>
      <width>99</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>库存处理</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="combo_split_button">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>80</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Combo拆分</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
